### ISSUE TITLE
Doesn't react on emits ue4.19 sio0.8.0 (and others)

### ISSUE THREAD

**Issue by Nestoro** [getnamo/socketio-client-ue4#90]
Hello,
this:
![untitled](https://user-images.githubusercontent.com/13397845/42170482-ea82b13a-7e16-11e8-8cb3-c2177988f4d5.png)

is not producing any output clientside...
when i began to use this plugin i tested this functionality and it worked. Somehow it stopped to do so but i can't figure out why...

The serverside snippet that reacts on the emits is:
```
socket.on('test',_ => {
                console.log('test');
                this.io.emit('test', null);
});
```
and the serverside logs spam (as intended).

this does affect all functions i implemented so far, this is just the most basic one i could come up with...
i also tested it in other projects, and with other versions of sio but it works nowhere also i used your server example from socketio-client-ue4-example to emit the responses to ensure it isn't my server but there i experience the same...

strange things happen here...
~Nestoro

EDIT:
To sum up: the On Event(SocketIO) is not getting triggered smh.

EDIT:
after a couple of more non resulting experiments i noticed a Warning that might be connected to the issue:
`LogUObjectGlobals: Warning: Failed to find object 'Class None.SIOJJsonObject'`

**Comment by Nestoro** [comment]
@getnamo 'Awaiting Replication'
are you? or shall i?

**Comment by Nestoro** [comment]
OK nvm,
while browsing other issues i found out that you got to bind events before they appear. I don't know how i got it running without that but now that i bind them it works again.
sorry for the, now stupid, question.....

**Comment by getnamo** [comment]
No question is stupid, glad it all worked out. The repo readme at https://github.com/getnamo/socketio-client-ue4 should be fairly thorough, but if you feel like more documentation is needed, kindly add to e.g. https://github.com/getnamo/socketio-client-ue4/wiki
